tests/modules/variantbam/test.yml

```diff
@@ -0,0 +1,7 @@
+- name: variantbam test_variantbam
+  command: nextflow run tests/modules/variantbam -entry test_variantbam -c tests/config/nextflow.config
+  tags:
+    - variantbam
+  files:
+    - path: output/variantbam/test.bam
+      md5sum: fc08f065475d60b3b06ee32920564d4b
```

Comment:
Missing the `qcreport`? Or does the test not generate one (as I  see you've specified it as optional)?

Reply:
Good catch, yes definitely should add test for that and implement it.